_field-types/supported-field-types/semantic.md

```diff
@@ -0,0 +1,239 @@
+
+**Optional: Controlling Semantic Field Processing**
+
+Behind the scenes, semantic field processing relies on system ingest processors. This behavior is controlled by a cluster setting. If disabled, semantic fields will no longer trigger automatic embedding generation or text chunking during ingestion or update.
+
+Disabling this setting will not cause ingestion to fail—documents will still be ingested successfully, but no embeddings will be generated or updated for semantic fields. By default, it's enabled and you can disable it through the below request:
+
+```json
+PUT _cluster/settings
+{
+  "persistent": {
+    cluster.ingest.system_pipeline_enabled: "false"
+  }
+}
+```
+{% include copy-curl.html %}
+
+### Text Chunking
+
+By default, text chunking is `disabled` for `semantic` fields. This is because enabling chunking requires storing each chunk’s embedding in a nested object, which can increase search latency. Searching nested objects involves joining child documents back to their parent, along with additional scoring and aggregation logic. The more matching child documents there are, the higher the potential latency.
+
+However, if you're working with long-form text and want to improve search relevance, you can enable chunking by configuring the semantic field accordingly when creating the index, as shown below:
+
+```json
+PUT /my-nlp-index
+{
+  "mappings": {
+    "properties": {
+      "passage": {
+        "type": "semantic",
+        "model_id": "nF7yX5cBsaYnPfyOq2SG",
+        "chunking": true
+      }
+    }
+  }
+}
+```
+
+### Limitations
+When using a `semantic` field with a dense model, OpenSearch automatically creates a `knn_vector` field to store the embeddings. The `dimension` and `space_type` are derived from the model configuration, so you must ensure that this information is available before using the model. Other `knn_vector` parameters use default values, and customization is not currently supported.
+
+To enable text chunking, you can set `chunking: true` in the semantic field configuration. However, chunking currently uses a fixed token-length algorithm by default, with no support for customizing the chunking strategy.
+
+For sparse models, OpenSearch applies a default prune ratio of `0.1` when generating sparse embeddings. This pruning configuration is currently not customizable. When querying a semantic field with a sparse model, using the [`neural_sparse_two_phase_processor`]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/neural-sparse-query-two-phase-processor/) to optimize search speed is not supported.
+
+Querying a semantic field from a remote cluster is not supported at this time.
+
+### Benchmark
+The introduction of the semantic field feature in 3.1.0 brings minimal impact on overall performance:
+
+#### Query Latency:
```

Comment:
Let's don't mention about latency. We can discuss about latency in the blog but not in document. Because the latency can be vary depending on how you run the test. For example, this 5% is true for only specific setup.